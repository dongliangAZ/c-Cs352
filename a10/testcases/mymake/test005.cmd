vgopts=$1    # Valgrind options
outfile=$2
errfile=$3
statusfile=$4

BASEDIR="/home/cs352/spring16/Assignments/assg10/testcases/./mymake"
rm -f _testTarg*
rm -f _testDep*
touch -t 160501011000.01 _testDep1
touch -t 160501011001.01 _testDep2
touch -t 160501011002.01 _testDep3

ulimit -t 10
valgrind $vgopts -q ./mymake ${BASEDIR}/mfile4 _testTarg1 >${outfile} 2>${errfile}
exitstatus=$?
echo $exitstatus > $statusfile
cat _testTarg1 >> ${outfile}

rm -f _testTarg*
rm -f _testDep*
