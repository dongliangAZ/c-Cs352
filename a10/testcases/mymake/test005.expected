echo "build test 1" > _testTarg1 ; touch -t 160501011100.00 _testTarg1
build test 1
